src/Gui/GraphvizView.cpp

```diff
@@ -146,6 +148,92 @@ class GraphvizWorker : public QThread {
     QByteArray str, flatStr;
 };
 
+// Simple wrapper around QGraphicsView to make panning possible
+class GraphVizGraphicsView final : public QGraphicsView
+{
+  public:
+    GraphVizGraphicsView(QGraphicsScene* scene, QWidget* parent);
+    ~GraphVizGraphicsView() = default;
+
+    GraphVizGraphicsView(const GraphVizGraphicsView&) = delete;
+    GraphVizGraphicsView(GraphVizGraphicsView&&) = delete;
+    GraphVizGraphicsView& operator=(const GraphVizGraphicsView&) = delete;
+    GraphVizGraphicsView& operator=(GraphVizGraphicsView&&) = delete;
+
+  protected:
+    void mousePressEvent(QMouseEvent *event) override;
+    void mouseMoveEvent(QMouseEvent *event) override;
+    void mouseReleaseEvent(QMouseEvent *event) override;
+
+  private:
+    bool   isPanning_;
+    QPoint panStart_;
```

Comment:
This is the matter of preference of course. I usually prefer if the member variables of a class are distinguished in some way, so that if I look at code, I immediately know from the name of the variable whether it was a parameter or belongs to the class.

But definitely, you are right, it is better to stick to the style of the surrounding parts, I removed the suffixes accordingly.

On a side note, I would try to avoid using `_` as a prefix, as it is reserved in the language.

Reply:
My personal preference is that code should be kept as uniform as possible but it's really difficult without a defined coding convention. So I'm trying to stick with the style with the scope, file, and/or module. It's not always obvious though :)